experimental/uefi/src/main.rs

```diff
@@ -0,0 +1,66 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#![no_main]
+#![no_std]
+#![feature(abi_efiapi)]
+#![feature(custom_test_frameworks)]
+#![test_runner(crate::test_runner)]
+#![reexport_test_harness_main = "test_main"]
+
+use uefi::{prelude::*, table::runtime::ResetType, ResultExt};
+
+#[entry]
+fn _start(_handle: Handle, mut system_table: SystemTable<Boot>) -> Status {
```

Comment:
Does this need to be named `_start` or is it just a convention? Either way, could you link to some reference material?

Reply:
Just a convention. I've added some comments.

Reply:
Great, thanks! This stuff is pretty new for everyone, including people familiar with "vanilla" Rust, so expect to have to over communicate / document things :)